app/controllers/dashboard/flickr_search_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Dashboard::FlickrSearchController < Dashboard::BaseController
+  respond_to :js
+
+  def search_flickr
+    @search_term = params[:search]
+
+    list = if @search_term.blank?
+      Rails.logger.warn("Search term is blank")
+      recent_photos
+    else
+      Rails.logger.warn("Search term provided")
+      search_with(@search_term)
+    end
+
+    Rails.logger.warn '++++++++++++++++++++++++++++++++'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.